demonstrations/tutorial_mol_geo_opt.py

```diff
@@ -0,0 +1,427 @@
+r"""
+Optimization of molecular geometries
+====================================
+
+.. meta::
+    :property="og:description": Find the equilibrium geometry of a molecule
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_spectra_h2.png
+
+.. related::
+   tutorial_quantum_chemistry Quantum Chemistry with PennyLane
+   tutorial_vqe Variational Quantum Eigensolver
+   tutorial_givens_rotations Givens rotations for quantum chemistry
+   
+*Author: PennyLane dev team. Last updated: 25 May 2021.*
```

Comment:
```suggestion
*Author: PennyLane dev team. Posted: 25 May 2021. Last updated: 25 May 2021.*
```
For the newer demos we've been adding posted dates as well.